.github/stale.yml

```diff
@@ -0,0 +1,62 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 30
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually,
+# but will remain marked as stale.
+daysUntilClose: 7
+
+# Issues or Pull Requests with these labels will never be considered stale.
+# Set to `[]` to disable
+exemptLabels:
+  - bug
+  - 'broken develop'
+  - 'good first issue'
+  - 'help wanted'
+  - hotfix
+  - regression
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue / pull request has been automatically marked as stale because it
```

Comment:
For the PRs I assume that keeping them rebased on top of `develop` is an activity. As long as the contributor of a PR takes the time to rebase it every ~30 days my understanding is that it won't be marked as stale.

If they are particularly important we can use a label to exempt them from this check or insert them in a Project or Milestone.

Reply:
Sorry for being imprecise: I wasn't trying to say that I'm against it for PRs, I was only saying that it will most likely not have a large effect on the number of PRs that we will have open :)